pytket/binders/circuit/Circuit/add_op.cpp

```diff
@@ -76,7 +70,8 @@ static Circuit *add_gate_method(
         "`condition_value` specified without `condition_bits`");
   }
   if (condition_given) {
-    py::object add_condition = condition_module().attr("_add_condition");
+    py::module condition = py::module::import("pytket.circuit.add_condition");
```

Comment:
do we know if this is slower than importing once only?

Reply:
I did some very informal timing of tests and didn't notice any significant difference. Under the hood this calls `PyModule_Import` which I think must be more or less a no-op if the module is already imported.